README.md

```diff
@@ -60,23 +59,30 @@ You can get the metadata in a collection as a Pandas DataFrame using:
 rows = coll.rows
 ```
 
+This data can then be modified or augmentated and added back to the collection using:
+
+```
+coll.replace_data(modified_rows)
+```
+
 You can get the images of a collection using:
 
 ```
 first_10_img_urls = coll.get_image_urls(list(range(10)))
 imgs = coll.download_image_batch(first_10_img_urls)
 ```
 
-If your collection supports the new multi-image-source functionality, you can see your available sources using:
+### Sources
+If a collection contains multiple image sources, these can be seen using:
 
 ```
 coll.show_sources()
 ```
 
-For source 2's (3rd in 0-indexed-list) images, you would use:
+Many operations require specifying which image source should be used. This can be specified by index or name for most functions.
 
 ```
-first_10_source3_img_urls = novo_col.get_image_urls(list(range(10)), source=2)`
+first_10_source3_img_urls = coll.get_image_urls(list(range(10)), source=2)
```

Comment:
Wondering if having 'source3' in the variable name and then source=2 could be confusing. Maybe if the example is with source=0, and variable name with 'source0', there is no doubt.